### ISSUE TITLE
RuntimeError: CUDNN_STATUS_EXECUTION_FAILED

### ISSUE THREAD

**Issue by NHW2017** [yanxp/MetaR-CNN#52]
Maybe you should update the version. The torch and CUDA used in this version are too old, and there are many problems in the implementation.

**Comment by renqiqi** [comment]
> Maybe you should update the version. The torch and CUDA used in this version are too old, and there are many problems in the implementation.

have you solved this problem?

**Comment by NHW2017** [comment]
No, I haven't solved this problem yet, if you run this program successfully, I hope you can give me a little help

**Comment by NHW2017** [comment]
> > Maybe you should update the version. The torch and CUDA used in this version are too old, and there are many problems in the implementation.
> 
> have you solved this problem?

No, I haven't solved this problem yet, if you run this program successfully, I hope you can give me a little help

**Comment by NHW2017** [comment]
> > Maybe you should update the version. The torch and CUDA used in this version are too old, and there are many problems in the implementation.
> 
> have you solved this problem?

This week, I finally ran the code successfully. My previous environment was Ubuntu 16.04/18.04 rtx2080ti cuda8.0 / 9.0 / 9.2. Yes, I almost tried it, but failed. I didn't list too many problems. This week, I ran the code easily under Ubuntu 18.04 gtx1070 cuda9.0, and then I tried ubuntu18.04 1050ti / 1080ti cuda9.0 / 9.2, all of which were successful. However, the previous problems still exist in 2080ti and 2070s

**Comment by renqiqi** [comment]
> > > Maybe you should update the version. The torch and CUDA used in this version are too old, and there are many problems in the implementation.
> > 
> > 
> > have you solved this problem?
> 
> This week, I finally ran the code successfully. My previous environment was Ubuntu 16.04/18.04 rtx2080ti cuda8.0 / 9.0 / 9.2. Yes, I almost tried it, but failed. I didn't list too many problems. This week, I ran the code easily under Ubuntu 18.04 gtx1070 cuda9.0, and then I tried ubuntu18.04 1050ti / 1080ti cuda9.0 / 9.2, all of which were successful. However, the previous problems still exist in 2080ti and 2070s

thank you!
After that, I did not continue the experiment.I will try it according to your tips.

**Comment by hanhao2019102935** [comment]
I used Colaboratory to test this code, at first I encountered the same problem with you, then I changed the default cuda10 to cuda9.0,but RuntimeError occurred: CUDNN_STATUS_EXECUTION_FAILED.Have you now completed the code?If so, could you tell me about your configuration environment and what I should do about it？

**Comment by pengyinxw** [comment]
> > > Maybe you should update the version. The torch and CUDA used in this version are too old, and there are many problems in the implementation.
> > 
> > 
> > have you solved this problem?
> 
> This week, I finally ran the code successfully. My previous environment was Ubuntu 16.04/18.04 rtx2080ti cuda8.0 / 9.0 / 9.2. Yes, I almost tried it, but failed. I didn't list too many problems. This week, I ran the code easily under Ubuntu 18.04 gtx1070 cuda9.0, and then I tried ubuntu18.04 1050ti / 1080ti cuda9.0 / 9.2, all of which were successful. However, the previous problems still exist in 2080ti and 2070s

Hi @NHW2017,
Did you following the other experiment requirements as the author suggested, except the cuda version? Thank you!
